--- src/kxgenerator.desktop.orig	2006-08-05 01:05:00.000000000 +0200
+++ src/kxgenerator.desktop	2006-08-05 18:05:37.457202880 +0200
@@ -4,7 +4,7 @@
 Exec=kxgenerator
 Icon=kxgenerator
 Type=Application
-Categories=System;Application;SystemSetup;
+Categories=Qt;KDE;SystemSetup;
 Comment=X Server Configuration Utilitty
 Comment[br]=Ur meziant eeun evit KDE
 Comment[ca]=Una simple aplicació per al KDE
